app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -224,12 +225,29 @@
     }
   }
 
+  @include govuk-media-query($until: tablet) {
+    .gem-c-feedback__option-list-item--useful,
```

Comment:
Can we style `.gem-c-feedback__option-list-item` instead of all the variations?

Reply:
have applied styling to `.gem-c-feedback__option-list-item` not variations